src/pages/programs/index.tsx

```diff
@@ -0,0 +1,170 @@
+              turn-key WiFi solution for shelters and transitional housing
+              facilities. ShelterConnect is proud to provide over 4,000 people a
+              year with daily internet access in over 20 local shelters and
+              resource centers.
+            </p>
+            <p>
+              We focus on identifying strategies to reduce the lifetime program
+              cost for wiring a shelter. The three primary costs are:
+            </p>
+            <ul>
+              <li>Equipment and licensing</li>
+              <li>Installation labor</li>
+              <li>Ongoing ISP charges</li>
+            </ul>
+            <p>
+              In 2019 we established a partnership with the City of San
+              Francisco Digital Equity team to deliver 1GB fiber/ISP capability
+              to future ShelterConnect locations at no cost. Working with
+              partners, ShelterConnect has implemented free WiFi solutions at
+              over 20 locations including the Star Hotel, Mission Hotel,
+              MSC-South, and ten locations managed by Larkin Street Youth
+              Services.
+            </p>
+          </>
+        }
+        image={{
+          url: shelterConnect,
+          alt: "Shelter Connect",
+        }}
+        ctaButtons={[
+          {
+            text: "DONATE",
+            internalLink: "https://www.sheltertech.org/donate/",
+          },
+          {
+            text: "PARTNER WITH US",
+            onClick: () => setPartnershipFormIsOpen(true),
+          },
+        ]}
+      />
+      <ProgramBlock
+        title="Community Representative Program"
+        subtitle="providing program insight and engagement with the community"
+        description={
+          <>
+            <p>
+              ShelterTech collaborates with Community Representatives who have
+              experienced homelessness to engage with volunteers, help build
+              program awareness, and gain insight to inform our programs and
+              products. Community Representative are key participants in the
+              following events:
+            </p>
+            <ul>
+              <li>
+                Datathons are public events where volunteers from the community
+                help verify data in the SF Service Guide
+              </li>
+              <li>
+                Outreach events in partnership with local frontline service
+                organizations help connect underserved communities with needed
+                resources
+              </li>
+            </ul>
+          </>
+        }
+        image={{
+          url: communityDevelopmentImg,
+          alt: "Community Development",
+        }}
+        ctaButtons={[
+          {
+            text: "DONATE",
+            internalLink: "https://www.sheltertech.org/donate/",
+          },
+        ]}
+      />
+      <Spacer heightDesktop="20px" heightMobile=".37px" />
```

Comment:
What's up with this .37px spacing on mobile? If it shows up in the Figma design, then I think we can assume that it wasn't intentional, and that we can just round this to 0.